packages/prettier-plugin-java/test/test-utils.js

```diff
@@ -69,22 +67,22 @@ function testRepositorySample(testFolder, command, args) {
     );
 
     javaSampleFiles.forEach(fileDesc => {
-      it(`prettify ${relative(samplesDir, fileDesc.path)}`, function() {
+      it(`Performs a stable formatting for <${relative(
+        samplesDir,
+        fileDesc.path
+      )}>`, function() {
         this.timeout(5000);
```

Comment:
Why 5000ms timeout?

Reply:
I just kept that from what it was before. @Shaolans may have an answer to why it was added in the first place?

Reply:
this may be needed for bigger tests, does not seem appropriate for small unit tests